src/zenml/metadata_stores/mysql_metadata_store.py

```diff
@@ -26,13 +26,13 @@
 class MySQLMetadataStore(BaseMetadataStore):
     """MySQL backend for ZenML metadata store."""
 
-    username: Optional[str]
-    password: Optional[str]
+    username: Optional[str] = None
+    password: Optional[str] = None
```

Comment:
These weren't implicitly set to None based on the Optional typing?

Reply:
I gave it a quick try, here is what I found.
```python
from pydantic import BaseModel
from typing import Optional

class OptionalClass(BaseModel):
    
    not_none: str
    maybe_none: Optional[str]
    
OptionalClass(not_none='This is set')
```

>>> OptionalClass(not_none='This is set', maybe_none=None)

So, it gets set to None if not set explicitly when constructing.

Reply:
ah, you're right! It is a `pydantic.BaseModel`! Should I remove the explicit `None`s? I think that for readability they are good anyways and leads to the same result.

Reply:
My personal opinion would be to leave them in, always better to be explicit!